Liuliu/list pulls, branches, releases api doc (#1656)
dolthub/docs: content/products/dolthub/api/database.md

{% endswagger %}

## List pull requests

> Here is an example of getting pull requests list on the `museum-collections` database using an [authorization token](authentication.md). The response of pull request list is paginated, you can use the next page token included in the response to retrieve the following page of pull requests.

Reviewer: ```suggestion
Here is an example of listing pull requests for the `museum-collections` database using an [authorization token](authentication.md). The pull request list is paginated, so you need to use the next page token included in the response to retrieve the following pages of pull requests.
```

{% endswagger %}

## List branches

> Here's an example of how to list branches in database `museum-collections` under the organization `dolthub` using an [authorization token](authentication.md).

Reviewer: ```suggestion
Here's an example of how to list branches in the database `museum-collections` under the organization `dolthub` using an [authorization token](authentication.md).
```

> Getting branches list requires authentication, so you must include this authorization header in your request. See the [Authentication](authentication.md) section for more details.

Reviewer: ```suggestion
Listing branches requires authentication, so you must include this authorization header in your request. See the [Authentication](authentication.md) section for more details.
```

{% swagger src="../../../.gitbook/assets/dolthub-api/createRelease.json" path="/{owner}/{database}/releases" method="post" %}
[createRelease.json](../../../.gitbook/assets/dolthub-api/createRelease.json)
{% endswagger %}

## List releases

> Here's an example of how to list releases in database `museum-collections` under the organization `dolthub` using an [authorization token](authentication.md).

Reviewer: ```suggestion
Here's an example of how to list releases in the database `museum-collections` under the organization `dolthub` using an [authorization token](authentication.md).
```

> Getting releases list requires authentication, so you must include this authorization header in your request. See the [Authentication](authentication.md) section for more details.

Reviewer: ```suggestion
Listing releases requires authentication, so you must include this authorization header in your request. See the [Authentication](authentication.md) section for more details.
```
